index.html

```diff
@@ -670,6 +671,14 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
         [=list/append=] <var>newEntry</var> to <var>tuple</var>'s
         <a>performance entry buffer</a>.
         </li>
+        <li><a>Queue the PerformanceObserver task</a>.</li>
+      </ol>
+    </section>
+    <section data-link-for="PerformanceObserver">
+      <h2>Queue the PerformanceObserver task</h2>
```

Comment:
This algorithm references `relevantGlobal`, which should be an explicit argument.

Reply:
Done.